spotinst/resource_spotinst_ocean_aks_np_test.go

```diff
@@ -524,6 +524,19 @@ func TestAccSpotinstOceanAKSNP_Scheduling(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "scheduling.#", "1"),
 					resource.TestCheckResourceAttr(resourceName, "scheduling.0.shutdown_hours.#", "1"),
 					resource.TestCheckResourceAttr(resourceName, "scheduling.0.shutdown_hours.0.is_enabled", "false"),
+					resource.TestCheckResourceAttr(resourceName, "scheduling.0.shutdown_hours.0.time_windows.0", "Fri:15:30-Sat:15:30"),
```

Comment:
if shutdown_hours.is_enabled is false then why time_windows is set, remove and test

Reply:
removed